aspnetcore/migration/22-to-30.md

```diff
@@ -456,6 +456,29 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+## Opt in to runtime compilation of Razor views
+
+In ASP.NET Core 2.2, support for runtime compilation of Razor views was included out-of-the-box when calling `AddMvc`. This allowed changes to be made to views at runtime and for those changes to be seen by refreshing the browser without needing to restart the app. It also enabled Razor views that didn't exist when the app was built to to be generated at runtime. The following code shows all that was needed before ASP.NET Core 3.0:
+
+```cs
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddMvc();
+}
+```
+
+In ASP.NET Core 3.0, runtime compilation of Razor views is an opt-in scenario. To enable the same support as found in ASP.NET Core 2.2, add a reference to the [Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation/) NuGet package and call `AddRazorRuntimeCompilation`:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddMvc()
+        .AddRazorRuntimeCompilation();
+}
+```
+
+For more information, see <xref:mvc/views/view-compilation>.
```

Comment:
The linked page doesn't really say much more than what's here now, but it still seems to me like the link should stick around.